frontend/packages/process-editor/src/utils/bpmn/StudioModeler.ts

```diff
@@ -44,6 +58,10 @@ export class StudioModeler {
     this.modeling.updateProperties(this.getElement(), { ...properties });
   }
 
+  public updateModdleProperties<T>(properties: T, element: ModdleElement): void {
+    this.modeling.updateModdleProperties(this.getElement(), element, { ...properties });
```

Comment:
Is it possible to test what CodeCov comments on in this file, or is it not done because of the bpmn package we use? 😄

Reply:
I added a comment at the top of the class with description. :)